static/js/settings_user_groups_legacy.js

```diff
@@ -36,6 +37,45 @@ export function reload() {
     populate_user_groups();
 }
 
+export function update_user_groups_settings_tip() {
```

Comment:
We can probably have a condition here to return if settings overlay is not opened.

Reply:
This function `update_user_groups_settings_tip()` only runs when `User Groups` is clicked in Organization Settings. Do we need the condition in this case also?

Reply:
This function `update_user_groups_settings_tip()` only runs when `User Groups` is clicked in Organization Settings. Do we need the condition in this case also? @sahil839

Reply:
It is also called in `server_events_dispatch.js` when event for change in `user_group_edit_policy` is recieved, right?